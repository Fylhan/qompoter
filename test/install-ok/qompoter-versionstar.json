{
	"name": "test/install-ok",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v1.0",
		"git/test": "v0.1.*"
	}
}
