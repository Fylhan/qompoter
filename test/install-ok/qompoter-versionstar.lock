{
  "name": "install-ok/qompoter-versionstar",
  "date": "2016-11-24T22:01:30+01:00",
  "require": {
    "fylhan/posibrain": {"version": "v0.2", "type": "git", "url": "", "md5sum": "f2be7bd1e5da767a9a9b0c8dc3c7aa31"},
    "cp/fylhan": {"version": "v1.1.3-beta", "type": "qompoter-fs", "url": "", "md5sum": "9d57702586a06dcd5de4d717d936c9d6"},
  }
}
