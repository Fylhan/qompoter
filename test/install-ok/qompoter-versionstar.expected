Qompoter
======== install

* fylhan/posibrain v0.2
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

* cp/fylhan v1.*
  Downloading sources...
  done

* fylhan/platphorm master
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

done
